inspectkv/inspectkv.go

```diff
@@ -67,8 +67,8 @@ func GetDDLInfo(txn kv.Transaction) (*DDLInfo, error) {
 	return info, nil
 }
 
-// GetDDLBgInfo returns DDL background information.
-func GetDDLBgInfo(txn kv.Transaction) (*DDLInfo, error) {
+// GetBgDDLInfo returns DDL background information.
+func GetBgDDLInfo(txn kv.Transaction) (*DDLInfo, error) {
```

Comment:
A little confused why rename GetDDLBgInfo to GetBgDDLInfo,  from your comment `DDL background information`, seems GetDDLBgInfo is clearer.